feature/system/gnmi/cliorigin/ate_tests/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -100,9 +101,16 @@ func TestCLIBeforeOpenConfig(t *testing.T) {
 		t.Fatalf("Could not resolve path: %v", errs)
 	}
 
+	resolvedTypePath := gnmi.OC().Interface(dp.Name()).Type().Config().PathStruct()
```

Comment:
I'd factor out a helper function resolveInterfacePathAndType that provides the functionality of lines 98-108, so it can be reused rather than duplicated between the test cases